client/src/Components/QA/CoffeeShopQuestions.js

```diff
@@ -120,23 +119,54 @@ const CoffeeShopQuestions = ({ questionsShopId }) => {
         ) : (
           <p>No more questions</p>
         )}
-      </StyledResultCard>
+      </div>
```

Comment:
can you explain why we revert this?

Reply:
Yeah no problem. It was just un-needed. Everything in StyledResaultCard was not in Figma so I undid it to get better spacing. Can show you in the morning